assets/src/block-templates/side-image-with-text-and-cta/template.js

```diff
@@ -0,0 +1,28 @@
+import mainThemeUrl from '../main-theme-url';
+
+const { __ } = wp.i18n;
+
+const template = ({
+  title = null,
```

Comment:
I think we should set an empty `string` instead of `null`. Why? 
First, because the `core/heading` has already defined the `content` attribute as a `string` ([see the WP heading doc](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/heading/block.json#L15)) and [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/929/files#diff-03eabc1163fecc06781194aaf556684fd1c6b304204ab2b21c638e0363469c5cR20) you are passing the content as `null`.
And finally to have more consistency with the attribute types.